cdp/webhook.py

```diff
@@ -0,0 +1,192 @@
+from collections.abc import Iterator
+
+from cdp.cdp import Cdp
+from cdp.client.models.create_webhook_request import CreateWebhookRequest
+from cdp.client.models.update_webhook_request import UpdateWebhookRequest
+from cdp.client.models.webhook import Webhook as WebhookModel
+from cdp.client.models.webhook import WebhookEventFilter, WebhookEventType, WebhookEventTypeFilter
+from cdp.client.models.webhook_list import WebhookList
+
+
+class Webhook:
+    """A class representing a webhook."""
+
+    def __init__(self, model: WebhookModel) -> None:
+        """Initialize the Webhook class.
+
+        Args:
+            model (WebhookModel): The WebhookModel object representing the Webhook.
+
+        """
+        self._model = model
+
+    def __str__(self) -> str:
```

Comment:
can we add a `__repr__` also and move these to the bottom after all of the business logic methods like we have in other classes.

Reply:
done, moved `__str__` and `__repr__` to the end.